src/renderer/components/+workloads-overview/overview.tsx

```diff
@@ -33,36 +33,77 @@ import { replicaSetStore } from "../+workloads-replicasets/replicasets.store";
 import { jobStore } from "../+workloads-jobs/job.store";
 import { cronJobStore } from "../+workloads-cronjobs/cronjob.store";
 import { kubeWatchApi } from "../../../common/k8s-api/kube-watch-api";
-import { clusterContext } from "../context";
 import { WorkloadsOverviewDetailRegistry } from "../../../extensions/registries";
 import type { WorkloadsOverviewRouteParams } from "../../../common/routes";
+import { makeObservable, observable, reaction } from "mobx";
+import { clusterContext } from "../context";
+import { NamespaceSelectFilter } from "../+namespaces/namespace-select-filter";
+import { Icon } from "../icon";
+import { TooltipPosition } from "../tooltip";
 
 interface Props extends RouteComponentProps<WorkloadsOverviewRouteParams> {
 }
 
 @observer
 export class WorkloadsOverview extends React.Component<Props> {
+  @observable loadErrors: string[] = [];
+
+  constructor(props: Props) {
+    super(props);
+    makeObservable(this);
+  }
+
   componentDidMount() {
     disposeOnUnmount(this, [
       kubeWatchApi.subscribeStores([
         podsStore, deploymentStore, daemonSetStore, statefulSetStore, replicaSetStore,
         jobStore, cronJobStore, eventStore,
       ], {
-        preload: true,
-        namespaces: clusterContext.contextNamespaces,
+        onLoadFailure: error => this.loadErrors.push(String(error)),
+      }),
+      reaction(() => clusterContext.contextNamespaces.slice(), () => {
```

Comment:
Why can't it just be `() => clusterContext.contextNamespaces`?

Reply:
Because then it only tracks assignments to the `clusterContext.contextNamespaces` field itself, and not internal changes.

Reply:
Ah, right. You have to "dot" into it. Ugh.